.eslintrc

```diff
@@ -1,11 +1,10 @@
 {
-  "extends": "rackt",
+  "extends": ["rackt", "plugin:react/recommended"],
```

Comment:
I've been using this config lately:
```json
{
  "parser": "babel-eslint",
  "env": {
    "browser": true,
    "node": true
  },
  "plugins": [
    "import",
    "react"
  ],
  "extends": [
    "eslint:recommended",
    "plugin:import/errors",
    "plugin:react/recommended"
  ],
  "rules": {
    "prefer-arrow-callback": 2,
    "semi": [ 2, "never" ]
  }
}
```

Regardless, we should probably remove the `rackt` extension, since that repo doesn't exist anymore.